circle.yml

```diff
@@ -30,6 +30,8 @@ dependencies:
 
 test:
   override:
+    # Check whether the committer forgot to run `go generate`.
+    - docker run cockroachdb/cockroach-dev shell "go generate ./... && git ls-files --modified --deleted | diff /dev/null -"
```

Comment:
That'll require some more changes since we add cockroach.sh in the directory, but I'll just move that to the root before I merge this.